### PR TITLE
Integrated RGB, Depth, PyRobot actuation noises, initial camera orientation.

### PR DESCRIPTION
**Pull request by mathfac**
- Integrated RGB noises
- Integrated Depth noise 
- Added PyRobot noisy actuations action space
- Integrated initial camera orientation
- Updated `objectnav_mp3d.yaml` config
- Improved `overwrite_config` to decrease code duplication
- Added fix to respect CONTENT_SCENES for habitat baselines training

## How Has This Been Tested
Added test testing all RGB, Depth, Actuation noises on Habitat API side. Checked that if disable any of the defined noises the test will fail, generated videos.

## Types of changes

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: habitat_baselines/common/env_utils.py [hunk 1]
```diff
@@ -54,8 +54,11 @@ def construct_envs(
     configs = []
     env_classes = [env_class for _ in range(num_processes)]
     dataset = make_dataset(config.TASK_CONFIG.DATASET.TYPE)
-    scenes = dataset.get_scenes_to_load(config.TASK_CONFIG.DATASET)
+    scenes = config.TASK_CONFIG.DATASET.CONTENT_SCENES
+    if "*" in config.TASK_CONFIG.DATASET.CONTENT_SCENES:
+        scenes = dataset.get_scenes_to_load(config.TASK_CONFIG.DATASET)
 
+    print(f"len(scenes): {len(scenes)}")
```
Comment (Skylion007) [inline habitat_baselines/common/env_utils.py]:
```suggestion
```

File: test/test_sensors.py [hunk 2]
```diff
@@ -319,3 +324,115 @@ def test_get_observations_at():
...
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL = "GaussianNoiseModel"
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL_KWARGS = habitat.Config()
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL_KWARGS.INTENSITY_CONSTANT = 0.5
+    config.SIMULATOR.DEPTH_SENSOR.NOISE_MODEL = "RedwoodDepthNoiseModel"
+
+    config.SIMULATOR.ACTION_SPACE_CONFIG = "pyrobotnoisy"
+    config.SIMULATOR.NOISE_MODEL = habitat.Config()
+    config.SIMULATOR.NOISE_MODEL.ROBOT = "LoCoBot"
+    config.SIMULATOR.NOISE_MODEL.CONTROLLER = "Proportional"
+    config.SIMULATOR.NOISE_MODEL.NOISE_MULTIPLIER = 0.5
```
Comment (erikwijmans) [inline test/test_sensors.py]: @mathfac These didn't get added into the default config for some reason?  Can you please add them.

Comment (mathfac) [reply test/test_sensors.py]: @erikwijmans because we don't use `NOISE_MODEL` by default.

**Comment by rosanom** [comment]
Hi,
I'm trying to test the `test_noise_models_rgbd()` function of the `test/test_sensors.py` file but I get this error:
<pre>line 389, in test_noise_models_rgbd
    env = habitat.Env(config=config, dataset=None)
...
...
TypeError: 'NoneType' object is not callable
</pre>
P.S. I just cloned the stable branch of the `habitat-api` repo and I noticed that the local `test/test_sensors.py` file was an old version, so I manually copied the new code from here to my local file. Could the problem be related to an old cloned version of the project?
Thank you

**Comment by rosanom** [comment]
Solved. The problem occurred because I cloned the stable branch, currently not up-to-date.

File: test/test_sensors.py (hunk 2 above)
Comment (Xiaoming-Zhao) [reply test/test_sensors.py:385]: Hi, I just traced down to the definition of the noise model [here](https://github.com/facebookresearch/habitat-sim/blob/master/habitat_sim/agent/controls/pyrobot_noisy_controls.py#L85) in `habitat-sim`. May I know whether the noise model utilized here is the same as described in Section 9 of [this paper](https://arxiv.org/pdf/1912.06321.pdf), which are fitted on real-world data? Thanks!

Comment (erikwijmans) [reply test/test_sensors.py:385]: Yes, this is the same noise model (note that it originates from https://github.com/facebookresearch/pyrobot#citation)

Comment (Xiaoming-Zhao) [reply test/test_sensors.py:385]: Got you. Thanks!

**Pull request merged**